### ISSUE TITLE
can't figure out how to switch between multi documents

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#480]
```
What steps will reproduce the problem?
1.can't figure out how to switch between multi documents
2.Multi-document support seem still unavailabe
3.

What is the expected output?
switching between mutli documents

What do you see instead?
can open one document only

What version of the product are you using?
EBookDroid 2 tech preview r2430  

On what operating system?
cm10.1

What device are you using?
nook hd
Please provide any additional information below.

text selection , annotation , and dictionary function all work great

```

Original issue reported on code.google.com by `wbhuan...@gmail.com` on 5 Apr 2013 at 3:58

**Comment by GoogleCodeExporter** [comment]
```
if title bar is visible, u may click a book title and see opened books and 
other views listed.
Also the recent view is shown if application icon clicked.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 5 Apr 2013 at 4:27
- Changed state: **Discussion**
- Added labels: **tech-preview**

**Comment by GoogleCodeExporter** [comment]
```
yes , just try it , fantastic , to switch between multi documents, turn on 
title bar first .
look forward to more annotation functions( such as comment and highlight)


```

Original comment by `wbhuan...@gmail.com` on 5 Apr 2013 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
Can you create a list of annotation tools you want to see? Right now we have 
pen with 3 width and optional pressure handling.
```

Original comment by `Andrei.K...@gmail.com` on 5 Apr 2013 at 6:16

**Comment by GoogleCodeExporter** [comment]
```
as an ebook reader , basic annotation tools should include highlight(or 
underline), comment(or text box), just in par with free adobe reader, 
additional features(such as those appear in ezpdf) should be goodies for 
donators.
```

Original comment by `wbhuan...@gmail.com` on 6 Apr 2013 at 1:49

**Comment by GoogleCodeExporter** [comment]
```
I do not understand what other highlight tool you are talking about.

How its behaviour should differ from current implementation?
```

Original comment by `Andrei.K...@gmail.com` on 6 Apr 2013 at 12:48

**Comment by GoogleCodeExporter** [comment]
```
 highlight  is just a standard annotation tool, first select the text you want to highlight, then apply the color you choose  on them , current annotation tool in ebookdroid   is called freehand in other  pdf  annotation app.
```

Original comment by `wbhuan...@gmail.com` on 6 Apr 2013 at 2:01

**Comment by GoogleCodeExporter** [comment]
```
Ok. I see. I think in next public release there will be highlight tool.
```

Original comment by `Andrei.K...@gmail.com` on 6 Apr 2013 at 8:13

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Too bad my devices stop at android 2.x
but as for annotations tools I would suggest
- underline
- striketrough
which can be derived as particular instances of highlighting with a thin black 
line.
And then at least 
- two types of 'transparent' highlighting (underline and full height) in at 
least 
- three different user defined colors that can be used at once. 

This will allow for six different coded highlighting schemes for titles, 
sections, formulas (in order to add structure to the text) and for highlighting 
key concepts, important paragraphs and definitions.
(I use dark green, light green and yellow and the effect is not as bad as one 
might at first imagine :-) )
```

Original comment by `peltio` on 7 Apr 2013 at 12:25

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
And if dreaming is allowed: 
one feature that would be priceless in highlighting formulae is a 'framed box' 
highlighting tool.
For example, I use to enclose important formulae (on real paper books) in a 
yellow box with a green frame. The tool should work like a 'draw rectangle 
tool', since formulae are usually two-dimensional entities spanning two or more 
lines.

Moreover, it would be nice for the 'straight line' highlighting tool to be able 
to adapt to slightly inclined text (slope detection in the proximity of the 
starting point?). This will help highlighting text on not perfectly aligned 
scanned documents.

Come to think of it... I guess the highlighter tool needs two different 
implementations:

- one for OCRed DJVU and PDFs (documents with accessible text) that would 
automatically align to the lines of text. This can specialize into underline, 
strikethrough and full highlight. User select the text and then choose the tool.

- one for scanned documents (pages are really images) that can produce a 
rectangular framed overlay with the same inclination as the page - slope 
detection can be applied to the bulk of the page). This can specialise into 
framed rectangles, highlighting rectangles, highlight full line, strikethrough, 
underline.
User select the area and then chooses the tool. the area selection should have 
the same inclination as the bulk of the page.

Never seen before. Anywhere.

I guess there is no way to change the text in a pdf or djvu file to add or 
remove italics and bold face, uhu?

(Talking of bold face, look what I've had the guts to ask you to do... :-) )


```

Original comment by `peltio` on 7 Apr 2013 at 12:59

**Comment by GoogleCodeExporter** [comment]
```
Thank you for detailed desciption of desired solution (as always). I try to 
implement as much as possible.
```

Original comment by `Andrei.K...@gmail.com` on 7 Apr 2013 at 8:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `Andrei.K...@gmail.com` on 17 May 2013 at 7:48
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.1: switching between documents are added in left side menu.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 12:14
- Changed state: **Fixed**
